packages/core/components/badge/index.js

```diff
@@ -1,40 +1,56 @@
 import React from "react";
 import PropTypes from "prop-types";
 import styled from "styled-components";
-import { borderRadius, themeGet } from "styled-system";
-import Text from "../text";
+import { themeGet } from "styled-system";
+import {
+  COMMON,
+  TYPOGRAPHY,
+  MISC,
+  LAYOUT,
+  BORDER,
+  BACKGROUND,
+  POSITION
+} from "../../constants";
 
-const StyledBadge = styled(Text)`
+const Badge = styled.span`
+  padding-top: ${themeGet("badge.space.pTop")};
```

Comment:
Probably out of scope for this PR, but is there a reason for using `pTop`, `pBottom`, etc, instead of the styled-system convention of `pt`, `pb`, etc?

Reply:
Good question. Will switch it out when I get to themeing it.